src/unicast_monit.c

```diff
@@ -39,6 +39,9 @@
 #include "tune.h"
 #include "rewrite.h"
 #include "autoconf.h"
+#ifndef _WIN32
+#include <sys/time.h>
+#endif
 #ifdef ENABLE_CAM_SUPPORT
```

Comment:
cam.h includes that, but when ENABLE_CAM_SUPPORT is off that header isn't included